map1.py

```diff
@@ -16,3 +16,4 @@
     fg.add_child(folium.Marker(location=[lt,ln],popup="quotes n' hai ",icon=folium.Icon(color='red')))
 map.add_child(fg)
 map.save("map1.html")
+print(" this is a test commit")

```

Comment:
i thinkk you should have to done something better instead of just putting a line